### DISCUSSION TITLE
Prowler auto archives security hub findings from previous prowler scan

### DISCUSSION THREAD

**Discussion by am-avios** [prowler-cloud/prowler#1625] [General]
Hi @toniblyx 

Say for example i run 
#prowler aws --categories secrets -S
and then 
#prowler aws --categories internet-exposed -S

I noticed that after running the second prowler scan the security hub findings from previous scan get auto archived in security hub (Security hub Record State goes from Active to Archived).

Can this behaviour be changed?

**Comment by toniblyx** [comment]
We will give it a look too, we have made some changes here and we need to reproduce it first. Moving it to issues.
